src/main/java/noel/helper/Tasklist.java

```diff
@@ -0,0 +1,250 @@
+            return;
+        }
+
+        if (startTimeFormat == null || endTimeFormat == null) {
+            System.out.println("Invalid format for start/end time!");
+            return;
+        }
+
+        Task taskToAdd = new Events(task, startDateFormat, startTimeFormat, endDateFormat, endTimeFormat);
+
+        if (checkFull()) {
+            System.out.println("Array is full!");
+        }
+
+        taskList.add(taskToAdd);
+
+        String addedMessageEnd = "Now you have " + taskList.size() + " tasks in the list.";
+        String updateAdd = addedMessageStart + "\n" +  taskToAdd + "\n" + addedMessageEnd;
+        printFunction(updateAdd);
+    }
+
+    public void addDeadline(String task, String endDate) {
+
+        LocalDate date = dateFormat(endDate);
+        if (date == null) {
+            System.out.println("Invalid date!");
+            return;
+        }
+
+        LocalTime time = timeFormat(endDate);
+        if (time == null) {
+            System.out.println("Invalid date!");
+            return;
+        }
+
+        Task taskToAdd = new Deadlines(task, date, time);
+
+        if (checkFull()) {
+            System.out.println("Array is full!");
+        }
+
+        taskList.add(taskToAdd);
+
+        String addedMessageEnd = "Now you have " + taskList.size() + " tasks in the list.";
+        String updateAdd = addedMessageStart + "\n" +  taskToAdd + "\n" + addedMessageEnd;
+        printFunction(updateAdd);
+    }
+
+    public void addToDo(String task) {
+
+        Task taskToAdd = new ToDos(task);
+
+        if (checkFull()) {
+            System.out.println("Array is full!");
+        }
+
+        taskList.add(taskToAdd);
+
+        String addedMessageEnd = "Now you have " + taskList.size() + " tasks in the list.";
+        String updateAdd = addedMessageStart + "\n" +  taskToAdd + "\n" + addedMessageEnd;
+        printFunction(updateAdd);
+    }
+
+    public void printTaskList(){
+        if (checkEmpty()) {
+            System.out.println("List is empty!");
+        } else {
+            String filler = "____________________________________________________________";
+            System.out.println(filler);
+            int maxLength = taskList.size();
+            for (int i = 0; i < maxLength; i++) {
+                System.out.println(i + 1 + ". " + taskList.get(i));
+            }
+            System.out.println(filler);
+        }
+    }
+
+    public List<String> getTaskAsList() {
+        List<String> linesToAppend = new ArrayList<>();
+        for (Task t:taskList) {
+            linesToAppend.add(t.toFileString());
+        }
+        return linesToAppend.subList(0, linesToAppend.size());
+    }
+
+    public void markAsDone(int index) {
+        taskList.get(index).markAsDone();
+    }
+
+    public void unMark(int index) {
+        taskList.get(index).unMark();
+    }
+
+    public void remove(int index) {
+        taskList.remove(index);
+    }
+
+    public int size() {
```

Comment:
Since this is a getter method for retrieving the size of the tasklist perhaps using `getSize()` would be a more appropriate name